client/README.rst

```diff
@@ -1,5 +1,56 @@
 Quipucords UI
 =============
 
-User interface for Quipucords. This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+User interface for Quipucords. This project was bootstrapped with `Create React App <https://github.com/facebookincubator/create-react-app>`_.
+
+Requirements
+------------
+Before developing for Quipucords UI, some basic guidelines:
+ * Your system needs to be running `NodeJS version 6+ <https://nodejs.org/>`_
+ * To run the mock API, you'll need to be running `Docker <https://docs.docker.com/engine/installation/>`_
```

Comment:
(stick with one way to say something to keep things parallel and like a formula, so...)
...your system needs to be running...
in both places